Bio/GenBank/__init__.py

```diff
@@ -466,10 +466,10 @@ def __init__(self, debug_level=0, use_fuzziness=1,
     def parse(self, handle):
         """Parse the specified handle.
         """
-        self._consumer = _FeatureConsumer(self.use_fuzziness,
-                                          self._cleaner)
-        self._scanner.feed(handle, self._consumer)
-        return self._consumer.data
+        _consumer = _FeatureConsumer(self.use_fuzziness,
```

Comment:
I wonder if any existing code relied on the private attribute? Your version does seem cleaner...